docs/content_structure.md

```diff
@@ -4,20 +4,73 @@
+- **Fields used**:
+  - Contribution Author (field_contribution_author): References the user who authored the contribution.
+  - Contribution Date (field_contribution_date): The date the contribution was made.
+  - Contribution Details (field_contribution_description): A long, formatted text field describing the contribution.
+  - Contribution Issue Link (field_code_contrib_issue_link): References the related issue content type for the contribution.
+  - Contribution Link (field_code_contrib_link): A URL link associated with the contribution.
+  - Contribution Project (field_code_contrib_project): References the project taxonomy term associated with the contribution.
+  - Contribution Type (field_contribution_type): References the contribution type from the Contribution Type taxonomy.
+  - Files Count (field_code_contrib_files_count): Number of files included in the contribution.
+  - Issue Status (field_code_contrib_issue_status): Text list field indicating the current status of the related issue.
+  - Moderator Comments (field_contrib_moderator_comment): Formatted text field for comments by the moderator.
+  - Patches Count (field_code_contrib_patches_count): Number of patches included in the contribution.
+  - Technology (field_contribution_technology): References the technology taxonomy term related to the contribution.
+
+### Event
+
+- **Description**: Use the Event content type to create and manage events, specifying details like date, location, and description for organizational or community activities.
+- **Fields used**: It has common event related fields (Active Event, Description, Event Additional Links, Event Address, Event Dates, Event Link, Event Location) and additional field named field_event_contrib_event_type. This field allows content to reference specific event types from the “Event Type” taxonomy vocabulary.
+
+### Event Contribution
+
+- **Description**: Use the Event contributions content type to document contributions made during events, including presentations, workshops, or collaborative efforts.
+- **Fields used**:
+  - Active Event (field_event_active): Indicates whether the event is currently active or inactive (Boolean).
+  - Description (body): A detailed, formatted text field with a summary to describe the event.
+  - Event Additional Links (field_event_additional_links): URLs for additional information or resources related to the event.
+  - Event Address (field_event_address): The physical address where the event will take place.
+  - Event Dates (field_event_dates): The start and end dates (and times) for the event.
+  - Event Link (field_event_link): A primary URL link associated with the event.
+  - Event Location (field_event_location): Geographical coordinates (latitude and longitude) of the event’s location.
+  - Event Type (field_event_contrib_event_type): References the type of event, categorized by terms from the Event Type taxonomy.
```

Comment:
All these are fields on the Event content type, not Event contribution.

Reply:
@hussainweb: It appears like that, but the reality is - those fields are practically under the Event Contribution content type.

Reply:
> How do you mean? When creating an event contribution, all you can do is select an existing event. You cannot touch any of the fields mentioned here when creating an event contribution.

@hussainweb: Thank you for the clarification and I made mistake by switching fields of Events with Event Contributions. Now, I fixed it. Please review.